### ISSUE TITLE
Number of digits allowed by default

### ISSUE THREAD

**Issue by ellenaua** [PolymerElements/gold-cc-cvc-input#43]
When we have these two inputs on the page

<gold-cc-input auto-validate card-type="{{cardType}}"></gold-cc-input>
<gold-cc-cvc-input card-type="[[cardType]]"></gold-cc-cvc-input>

user sometimes wants to start typing from second one. But that allows only 3 digits by default (when cardType is unknown). Maybe it makes sense to allow both 3 and 4 symbols when cardType is undefined yet?

**Comment by notwaldorf** [comment]
Yeah, that makes sense! Would you be interested in sending a PR and an unit test to add this feature?

**Comment by ellenaua** [comment]
@notwaldorf PR is here https://github.com/PolymerElements/gold-cc-cvc-input/pull/47 but please check if that's ok
